veterans/components/homePage/initiatives/map/Map.tsx

```
@@ -0,0 +1,33 @@
+import { regions } from 'icons/regions';
+import st from './Map.module.css';
+import InitiativeCount from './InitiativeCount';
+
+export default function Map() {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)